src/ilcdlib/epd/test_epd_reader.py

```diff
@@ -58,6 +60,31 @@ def test_read_basic_fields_industry_epd(self):
         self.assertEqual(self.epd_reader_industry.get_program_operator_id(), "EPD-HAM-20220202-ICD1-DE")
         self.assertEqual(self.epd_reader_industry.get_date_published(), datetime.date.fromisoformat("2022-10-10"))
 
+    def test_read_basic_fields_scenario_epd(self):
+        self.assertEqual(
+            self.epd_reader_industry.get_scenario_names(self.LANG), {'S1': '100% recycling', 'S2': 'Scenario 2'}
+        )
+
+    def test_read_epditaly_fields_scenario(self):
```

Comment:
I would prefer to move Dialect specific test cases in separate test suites and keep just common logic in the main one.